### PR TITLE
beep boop

### PR DESCRIPTION
**Pull request by mid-kid**

### CODE REVIEW

File: docs/bugs_and_glitches.md [hunk 1]
```diff
@@ -1334,6 +1335,58 @@ This is a mistake with the “`…`” tile in [gfx/battle/hp_exp_bar_border.png
 ![image](https://raw.githubusercontent.com/pret/pokecrystal/master/docs/images/hp_exp_bar_border.png)
 
 
+## Move selection menu doesn't handle joypad properly
```
Comment (Rangi42) [inline docs/bugs_and_glitches.md]:
```
([Video](https://www.youtube.com/watch?v=vjFUo6Jr4po&t=438))
```

File: engine/pokedex/pokedex_2.asm [hunk 2]
```diff
@@ -124,14 +124,16 @@ DisplayDexEntry:
 	jr z, .skip_height
 	push hl
 	push de
+; Print the height with two of the four digits will be in front of the decimal point
```
Comment (Rangi42) [inline engine/pokedex/pokedex_2.asm]: Grammar: "Print the height, with two of the four digits in front of the decimal point."

File: engine/pokedex/pokedex_2.asm [hunk 3]
```diff
@@ -148,11 +150,12 @@ DisplayDexEntry:
 	or d
 	jr z, .skip_weight
 	push de
+; Print the height with four of the five digits will be in front of the decimal point
```
Comment (Rangi42) [inline engine/pokedex/pokedex_2.asm]: Grammar: "Print the height, with four of the five digits in front of the decimal point."

**Comment by Rangi42** [comment]
Since this misc-change PR is open anyway: please change the `ld a, $6` and `ld a, $7` to `ld a, $06` and `ld a, $07` in engine/pokegear/pokegear.asm:Pokedex_GetArea.PlaceString_MonsNest, for consistency with the other two town-map-border-drawing routines. (Tile IDs in general are written with two digits.)

**Comment by Rangi42** [comment]
Also there are some values that should be based on `SCREEN_WIDTH` in the steps here: https://github.com/pret/pokecrystal/wiki/Edit-the-Town-Map#6-remove-the-gray-border-for-more-map-area

**Comment by Rangi42** [comment]
One more change request: `end` should be `return` in the "Clair can give TM24 Dragonbreath twice" bugfix. All your other changes look good.

**Comment by Rangi42** [comment]
@mid-kid Please just make that `end`->`return` change and I can merge this.

**Comment by mid-kid** [comment]
Sorry, there you go.

**Pull request merged**